client/src/components/Unfollower.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import "./Unfollower.css";
+
+export default function Unfollower({ user }) {
```

Comment:
The mismatch that I also mentioned in my pull request is a problem. I solved this in backend but you chose frontend. Both works 💯

Reply:
Yep, we can develop a strategy for choosing only the Backend or Frontend side to solve the inconsistent object models like this one after discussing the next session 👍